build/charts/antrea/conf/antrea-agent.conf

```diff
@@ -193,37 +193,45 @@ apiPort: {{ .Values.agent.apiPort }}
-# <HOST> to <Service namespace>/<Service name>. For example,
-# "flow-aggregator/flow-aggregator" can be provided to connect to the Antrea
-# Flow Aggregator Service.
-# If PORT is empty, we default to 4739, the standard IPFIX port.
-# If no PROTO is given, we consider "tls" as default. We support "tls", "tcp" and
-# "udp" protocols. "tls" is used for securing communication between flow exporter and
-# flow aggregator.
-flowCollectorAddr: {{ .Values.flowCollector.collectorAddr | quote }}
-
-# Provide flow poll interval as a duration string. This determines how often the
-# flow exporter dumps connections from the conntrack module. Flow poll interval
-# should be greater than or equal to 1s (one second).
-# Valid time units are "ns", "us" (or "µs"), "ms", "s", "m", "h".
-flowPollInterval: {{ .Values.flowCollector.flowPollInterval | quote }}
-
-# Provide the active flow export timeout, which is the timeout after which a flow
-# record is sent to the collector for active flows. Thus, for flows with a continuous
-# stream of packets, a flow record will be exported to the collector once the elapsed
-# time since the last export event is equal to the value of this timeout.
-# Valid time units are "ns", "us" (or "µs"), "ms", "s", "m", "h".
-activeFlowExportTimeout: {{ .Values.flowCollector.activeFlowExportTimeout | quote }}
 
-# Provide the idle flow export timeout, which is the timeout after which a flow
-# record is sent to the collector for idle flows. A flow is considered idle if no
-# packet matching this flow has been observed since the last export event.
-# Valid time units are "ns", "us" (or "µs"), "ms", "s", "m", "h".
-idleFlowExportTimeout: {{ .Values.flowCollector.idleFlowExportTimeout | quote }}
+flowExporter:
+    {{- with .Values.flowExporter }}
+    # Enable FlowExporter, a feature used to export polled conntrack connections as IPFIX flow records from each agent
+    # to a configured collector. To enable this feature, you need to set "enable" to true, and ensure that the
+    # FlowExporter feature gate is also enabled.
+    enable: {{ .enable }}
+    # Provide the IPFIX collector address as a string with format <HOST>:[<PORT>][:<PROTO>].
+    # HOST can either be the DNS name, IP, or Service name of the Flow Collector. If
+    # using an IP, it can be either IPv4 or IPv6. However, IPv6 address should be
+    # wrapped with []. When the collector is running in-cluster as a Service, set
+    # <HOST> to <Service namespace>/<Service name>. For example,
+    # "flow-aggregator/flow-aggregator" can be provided to connect to the Antrea
+    # Flow Aggregator Service.
+    # If PORT is empty, we default to 4739, the standard IPFIX port.
+    # If no PROTO is given, we consider "tls" as default. We support "tls", "tcp" and
+    # "udp" protocols. "tls" is used for securing communication between flow exporter and
+    # flow aggregator.
+    flowCollectorAddr: {{ .collectorAddr | quote }}
```

Comment:
there is a small inconsistency between the Helm value path (`flowExporter.collectorAddr`) and the configuration parameter path (`flowExporter. flowCollectorAddr`). Should we rename the configuration parameter for consistency between the 2?

Reply:
Thanks for reviewing. I've renamed it to flowCollectorAddr.
network-flow-visibility.md is updated as well.